js/qg-main/qg-utils.js

```diff
@@ -4,10 +4,55 @@
     // Workaround to push columns - some pages on Matrix have a 3 col layout but there is no third columns
     // example - https://www.qld.gov.au/about/events-awards-honours/awards/reconciliation-awards/2015-award-winners
     function colPush(){
-        let $qg = $('#qg-three-col');
+        var $qg = $('#qg-three-col');
         if ($qg.find('#qg-section-nav').length <= 0) {
             $qg.addClass('no-secondary-nav');
         }
     }
     colPush();
+    // This function equal height of cards in a group, if it finds a class '.qg-cards__equal-height and row classes i.e row-1 , row-2 etc'.
+    var $container = $('.qg-cards__equal-height');
+    var gridType = function (){
+        var fgridType;
+        if ($container.find('.col-lg-4').length > 0){
+            fgridType = 3;
+        } else if ($container.find('.col-lg-6').length > 0){
+            fgridType = 2;
+        }
+        return fgridType;
+    };
+    var cardCount = function(){
+        return $container.find('.qg-card').length;
+    };
+    function setHeight() {
+        if ($container.length > 0) {
+            var loopCount = Math.ceil(cardCount() / gridType());
+            $('.qg-cards.qg-cards__equal-height').each(function () {
+                // Cache the highest
+                // Select and loop the elements you want to equalise
+                for (let i = 0; i <= loopCount; i++){
```

Comment:
Looping from 0 to `loopCount` inclusive will mean the number of iterations is `loopCount + 1`. Is that intended?

Reply:
yes we need to loop through all the columns to equal height

Reply:
updated loop count